pkg/cli/token.go

```diff
@@ -19,26 +19,33 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
+	"strings"
 	"time"
 
-	"github.com/golang-jwt/jwt"
 	"github.com/google/uuid"
+	"github.com/lestrrat-go/jwx/v2/jwa"
+	"github.com/lestrrat-go/jwx/v2/jwt"
 	"github.com/spf13/cobra"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 	"google.golang.org/grpc/credentials/insecure"
 	"google.golang.org/grpc/credentials/oauth"
 	"google.golang.org/protobuf/types/known/durationpb"
 
-	jvsapis "github.com/abcxyz/jvs/apis/v0"
+	jvspb "github.com/abcxyz/jvs/apis/v0"
 	"github.com/abcxyz/jvs/pkg/idtoken"
 )
 
+const (
+	Issuer  = "jvsctl"
+	Subject = "jvsctl"
+)
+
 var (
-	tokenExplanation string
-	breakglass       bool
-	ttl              time.Duration
-	issTimeUnix      int64
+	flagTokenExplanation string
```

Comment:
This is a best practice we should try to codify somewhere. Flag names in CLIs should be prefixed with `flag` to differentiate them from other vars.